README.md

```diff
@@ -22,18 +22,18 @@ Methods should look like this:
 ```objective-c
 - (void)doSomethingWithString:(NSString *)theString
 {
-    ...
+...
 }
 ```
 
 If you have more than one parameter, giving each its own line is preferred. If multiple lines are used, align each using the colon before the parameter.
 
 ```objective-c
 - (void)doSomethingWith:(GTMFoo *)theFoo
-                   rect:(NSRect)theRect
-               interval:(float)theInterval
+rect:(NSRect)theRect
```

Comment:
It looks like this and a few other lines' indentations were accidentally changed. Were these intentional?